src/components/carousel/index.stories.mdx

```diff
@@ -0,0 +1,405 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+import Carousel from ".";
+import Image from "../image";
+import Button from "../button";
+import Icon from "../icon";
+
+export const SampleCard = ({ src }) => (
+	<div>
+		<h3 style={{ padding: "0 0.25rem" }}>
+			<Image src={src} />
+		</h3>
+	</div>
+);
+
+export const SampleNextArrow = ({ onClick }) => (
+	<Button
+		style={{
+			position: "absolute",
+			top: "50%",
+			right: "-25px",
+		}}
+		accessibilityLabel="Next"
+		onClick={onClick}
+	>
+		<Icon name="ArrowRight" width="24" height="24" />
+	</Button>
+);
+
+export const SamplePrevArrow = ({ onClick }) => (
+	<Button
+		style={{
+			position: "absolute",
+			top: "50%",
+			left: "-25px",
+		}}
+		accessibilityLabel="Previous"
+		onClick={onClick}
+	>
+		<Icon name="ArrowLeft" width="24" height="24" />
+	</Button>
+);
+
+<Meta title="Components/Carousel" component={Carousel} />
+
+# Carousel
+
+A `<Carousel>` is a container for child components that may be viewed in sequence,
+forward or backward. The container can take in markup of children div containers. Here is a simple example:
+
+```jsx
+<div>
+	<h3>
```

Comment:
We want to make sure we are providing the best practise examples - using `<h3>` here is not a good example, and is a heading even needed?